helm/files/indexd.conf

```diff
@@ -0,0 +1,24 @@
+<VirtualHost *:80>
+    ServerName indexd.{{.Release.Namespace}}.svc.cluster.local
+
+    Include /etc/apache2/sites-available/logs.conf
+    LogLevel warn
+    ErrorLog /dev/stderr
+    CustomLog /dev/stdout f5_forwarder
```

Comment:
Did you mean to flip this back to f5_forwarder? I don't mind either way given that I think we still have to set up a few more things to enable log aggregation from K8s, I just want to confirm.

Reply:
I did mean to flip it, since we are now relying on the log format from the base container. We can probably override if needed in the future.

Reply:
Okay, makes sense.